templates/solutions/ai-infrastructure/_form-ai-infrastructure.html

```diff
@@ -0,0 +1,167 @@
+<div class="p-modal" style="display: none;" id="ai-infrastructure-modal">
+  <div class="p-modal__dialog is-paper" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <div class="row--50-50 p-strip is-shallow">
+          <div class="col">
+            <h2 class="p-heading--1">Contact Canonical</h2>
+            <p>Fill in this form and a member of our team will be in touch shortly.</p>
+          </div>
+          <div class="col">
+            <button class="p-modal__close p-modal-close-button "
+                    aria-controls="ai-infrastructure-modal"
+                    aria-label="Close active modal"
+                    style="margin-top: 0">Close</button>
+            <form action="https://ubuntu.com/marketo/submit"
+                  method="post"
+                  class="modal-form">
+              <div class="p-section--shallow">
+                <label for="firstName" class="is-required">First name:</label>
+                <input required id="firstName" name="firstName" maxlength="255" type="text" />
+              </div>
+              <div class="p-section--shallow">
+                <label for="lastName" class="is-required">Last name:</label>
+                <input required id="lastName" name="lastName" maxlength="255" type="text" />
+              </div>
+              <div class="p-section--shallow">
+                <label for="email" class="is-required">Email:</label>
+                <input required
+                       id="email"
+                       name="email"
+                       maxlength="255"
+                       type="email"
+                       pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </div>
+              <div class="p-section--shallow">
+                <label for="country">Country:</label>
```

Comment:
I think we should use the existing partial for this field, example [here](https://github.com/canonical/canonical.com/blob/f8ec590bdaa74681621c82fce2023ddb95c114a3/templates/solutions/secure-open-source/_form-secure-open-source.html#L26)